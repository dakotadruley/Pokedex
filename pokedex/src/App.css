body {
  font-family: monospace;
}

h1 {
  font-size: 50px;
}

td {
  border-width: 0.25em;
  border-style: solid;
  border-color: black;
  font-size: 15px;
}

.poke {
  max-width: 600px;
  border-radius: 15px;
  border-style: solid;
  border-color: black;
}

.imageContainer {
  text-align: center;
}

ul {
  list-style: none;
}

th {
  font-size: 30px;
}

.pok√©s {
  display: flex;
  flex-direction: row;
}