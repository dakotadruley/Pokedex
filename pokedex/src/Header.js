import React, { Component } from 'react';

export default class Header extends Component {
    render() {
        return (
        <header>
            <div>
                <h1>Pok√©dex</h1>
            </div>
        </header>
        )
    }
};
