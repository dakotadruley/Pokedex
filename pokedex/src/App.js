import React, { Component } from 'react';
import './App.css';
import Header from './Header.js';
import PokeList from './PokeList.js';


export default class App extends React.Component {
  
  render () {
    return (
      <>
      <Header />
      <div>
      </div>
      <PokeList />
      </>
    )
}}
