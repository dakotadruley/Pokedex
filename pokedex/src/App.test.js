import React from 'react';
import App from './App';
import Header from './Header.js';
// import PokeItem from './PokeItem';
// import PokeList from './PokeList';
import renderer from 'react-test-renderer';

test('renders App.js correctly', () => {
  const tree = renderer
    .create(<App />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

test('renders Header.js correctly', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

// test('renders PokeItem.js correctly', () => {
//   const tree = renderer
//     .create(<PokeItem array={[]}/>)
//     .toJSON();
//   expect(tree).toMatchSnapshot();
// });
// Learn how to pass props into tests