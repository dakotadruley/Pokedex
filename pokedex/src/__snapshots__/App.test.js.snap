// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App.js correctly 1`] = `
Array [
  <header>
    <div>
      <h1>
        Pokédex
      </h1>
    </div>
  </header>,
  <div />,
  <ul
    className="pokes"
  />,
  "loading",
]
`;

exports[`renders Header.js correctly 1`] = `
<header>
  <div>
    <h1>
      Pokédex
    </h1>
  </div>
</header>
`;
